stable/aws-node-termination-handler/values.yaml

```diff
@@ -4,7 +4,7 @@
 
 image:
   repository: amazon/aws-node-termination-handler
-  tag: v1.0.0
+  tag: v1.1.0
```

Comment:
Ah, I see. This PR is currently blocking https://github.com/aws/aws-node-termination-handler/pull/41

Reply:
yep, sorry should have referenced